rustls/src/msgs/handshake.rs

```diff
@@ -1042,6 +1042,11 @@ impl ClientHelloPayload {
         self.find_extension(ExtensionType::EarlyData)
             .is_some()
     }
+
+    pub fn has_cookie_extension(&self) -> bool {
```

Comment:
Not really a fan of having functions like this that only have one caller. Better to just inline it into the caller.

Reply:
I tend to agree, but this is prevailing style, and I'd prefer to address all of these together in a separate PR if we decide we should go that way.

Reply:
I don't agree. For me, functions like this exist to make the caller more readable. It's not only about whether there are more than one caller.